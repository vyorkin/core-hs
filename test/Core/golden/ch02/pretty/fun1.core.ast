[(Name "const",[Name "x",Name "y"],EVar (Name "x")),(Name "compose",[Name "f",Name "g"],ELam [Name "x"] (EAp (EVar (Name "f")) (EAp (EVar (Name "g")) (EVar (Name "x")))))]
