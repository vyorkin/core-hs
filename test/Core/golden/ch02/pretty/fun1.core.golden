const x y = x
compose f g = \ x. f (g x)